### DISCUSSION TITLE
Border color is wrong

### DISCUSSION THREAD

**Discussion by Eremmmm** [dotnet/maui#18455] [Q&A]
Hi, I started developing applications with net maui with .net 7.0 but I'm having a problem with the color of my border, in fact it is slightly lighter than its real color and I don't understand why. Do you have an idea? The color is the same as that of the label however it does not have the same shade. 

Screen on Windows plateform :
![image](https://github.com/dotnet/maui/assets/49162612/62cceab9-7394-4399-8b58-209e652d2d2c)

Border xaml :
```xml
                <Border
                    Stroke="{StaticResource DarkMainColor}"
                    StrokeThickness="2">
                    <Border.StrokeShape>
                        <RoundRectangle CornerRadius="2" BackgroundColor="{StaticResource DarkMainColor}"/>
                    </Border.StrokeShape>
                    <Entry Placeholder="John"
                           PlaceholderColor="{ StaticResource DarkMainColor}" 
                           TextColor="{ StaticResource DarkMainColor}" 
                           FontFamily="ArchivoRegular" 
                           WidthRequest="220" 
                           HeightRequest="50" />
                </Border>
```

**Comment by Vroomer** [comment]
Does the control have a focus? WinUI seems to have highlight border, which might alter your color. It could also be result of anti-aliasing. Do you see the same issue if you increase the border thickness?

**Reply by Eremmmm** [reply]
My control does not have a focus. But actually when I increase the thickness of the border the color still poses a problem so it seems to be an antialiasing problem. How to fix it?

![image](https://github.com/dotnet/maui/assets/49162612/59828b6d-bdda-455a-8163-74f2bd8f4a77)

P.S : I tried on Android and didn't see the problem.

**Reply by Vroomer** [reply]
Well if it's really crucial to match the color, I would first check if the problem with anti-aliasing is only on your device. If not, the obvious solution is to make the border slightly darker so that it visually matches. I'm afraid there is no option in WinUI to change how things are rendered.

**Reply by Eremmmm** [reply]
I'm actually going to try to do something different but it could be a bug, right?

**Reply by MartyIX** [reply]
Is it possible that the bug is this one https://github.com/dotnet/maui/issues/19356?

**Reply by Eremmmm** [reply]
It looks like it but I found it recently : https://github.com/dotnet/maui/issues/21087

**Reply by MartyIX** [reply]
> It looks like it but I found it recently : https://github.com/dotnet/maui/issues/21087

That's filed by ... me.

**Reply by Eremmmm** [reply]
I didn't pay attention.
